### PR TITLE
[iOS] Added missing features

### PR DESCRIPTION
**Pull request by mpetrenco**
### Changelog:
- Added missing `isBankingAppAvailable` method;
- Added missing `fetchTransactionDetails` method and sample app option;
- Added documentation to the JudoPay class;

### CODE REVIEW

**Comment by marcghelas** [comment]
what is _'Get Transaction Details' feature_? I don't recall anything like that on the native side

File: JudoPay.tsx [hunk 1]
```diff
@@ -58,14 +58,62 @@ export type { JudoPBBAConfiguration } from './types/JudoPBBATypes'
...
+    //------------------------------------------------------------------
+
+    /**
+     * A method used to verify if any Pay By Bank App supported banking apps are installed
+     * on the device.
+     *
+     * This can be useful for conditionally rendering the PBBA button only if the bank app
+     * is available (PBBA won't work otherwise).
+     */
+    public isBankingAppAvailable(): boolean {
```
Comment (marcghelas) [inline JudoPay.tsx:104]: Should this method also be async and return Promise<boolean> since that's the only way to communicate with native?

Comment (mpetrenco) [reply JudoPay.tsx:104]: D:
You're probably right

Comment (marcghelas) [reply JudoPay.tsx:104]: Although this still works if you call the method like this:
`isAvailable = await judo.isBankingAppAvailable()`
so not sure.

**Review by marcghelas — APPROVED** [review]

File: sample-app/screens/TokenPayments/TokenPaymentsProps.tsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { JudoConfiguration, JudoAuthorization } from "judo-react-native"
 
 export default interface TokenPaymentProps {
+    navigation: any,
```
Comment (stpopa) [inline sample-app/screens/TokenPayments/TokenPaymentsProps.tsx:4]: can we avoid using `any` please?

**Review by stpopa — APPROVED** [review]

**Pull request merged**